src/cache.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+
+// TODO: move to its own package
+
+import fs from 'fs-extra'
+import moment from 'moment'
+
+export default class {
+  static async fetch (cachePath: string, cacheDuration: number, options: any = {}): Promise<?Array<?string>> {
```

Comment:
@RasPhilCo could you make the options a type since it is just `cacheFn` currently?